src/model.ts

```diff
@@ -12,6 +12,23 @@ export const createDefaultValidationResult = (): ValidationResult => ({
   message: '',
 });
 
+export interface FormFieldError {
+  key: string;
+  validationResult: ValidationResult;
+}
+
+export interface FormValidationSummary {
```

Comment:
FormValidationResult?

Reply:
Not sure here, is the summary of validating all fields plus record validations.

I think FormValidationResult would make sense because we are using FieldValidationResult

I will refactor it